common/src/main/java/com/wynntils/services/mapdata/providers/MapDataProvider.java

```diff
@@ -19,4 +19,8 @@ public interface MapDataProvider {
     Stream<MapIcon> getIcons();
 
     void onChange(Consumer<MapDataProvidedType> callback);
+
+    default void reloadData() {
```

Comment:
I think we should force implementors to think about reloading, i.e. it should not have a default implementation.

Reply:
Okay.